networkx/algorithms/approximation/tests/test_maxcut.py

```diff
@@ -0,0 +1,86 @@
+import random
+
+import networkx as nx
+import networkx.algorithms.approximation as a
```

Comment:
`a` is not a very descriptive alias. How about `from networkx.algorithms.approximation import maxcut`, which would require updating all of the `a.maxcut` lines to just `maxcut`.

Reply:
You are right. We blindly copied this from test_matching.